src/client/lobby/User/Users.css

```diff
@@ -1,16 +0,0 @@
-
```

Comment:
It's probably better to keep styles for separate components in separate files. You should try to all the user styles back in at some point